### DISCUSSION TITLE
Where to edit containers.conf (/usr/share/containers/containers.conf gets overwritten by update)

### DISCUSSION THREAD

**Discussion by chrismaster** [containers/podman#23503] [Q&A] [answered]
Where should a user (root) edit containers.conf for the system wide settings?
ATM I use /usr/share/containers/containers.conf , but an update on fedora reverted my pasta_options back to 
# pasta_options =[]
Is /etc/containers/containers.conf the right place?
This file is missing and the docs say it should also do the job.
Thx
c.

**Comment by rhatdan** [comment, marked as answer]
Yes. administrators should put the config in /etc/containers/containers.conf.
